### ISSUE TITLE
Accessibility issues for Past Appointments page

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#16583]
## What happened?
1. Android:  Unable to select a date range while Talkback is enabled in Android. It just says "Appointments Screen" when tapping on the selector picker.  

2. Android: Unable to select a date range while Talkback is enabled in Android and using bluetooth keyboard.  Can tab to picker but can not activate it with keyboard.

3. Android: "Hey Google" (using Voice Access app) will not click / activate the picker.
3. a. Voice control in iOS also will not activate / open the date ranger picker.

4. iOS:  When selecting the date picker, iOS Voiceover does not read what the picker currently displays.

5. iOS:  Voice Control does not choose from the date picker.  I try all sorts of commands by voice and it will not open. Upcoming/Past toggle operates via voice control just fine.

6. Android: The text "Select a date range" above the picker is not read on the Android. (Maybe related to #1 above?)

## Specs:
- Device: Galaxy Note 9 (latest update on phone software), iPhone SE (iOS 14.2)
- TalkBack / VoiceOver turned on.
- Bluetooth keyboard

## Steps to Reproduce
1. Android: Enable Talkback and navigate to Past Appointments screen.  Try to select from the date picker.

2. Android: Attach bluetooth keyboard. Enable Talkback and navigate to Past Appointments screen. Try to select from the date picker.

3. Android: Enable Voice Access app, "Hey Google tap [choose number by date picker]"

4. iOS: Enable VoiceOver and select the date picker. Nothing is spoken about the picker.

5. iOS: Enable Voice Control. Try to choose/open the date picker. Say "Tap Upcoming" or "Tap Past" to verify it is working and those toggles will be chosen.

6. Android: Enable TalkBack and proceed to try to highlight the text "Select a date range" above the picker.

## Desired behavior
User is able to select a date range from picker using TalkBack or VoiceOver.
User is able to select a date range from picker using Voice Access (Android) or Voice Control (iOS)
All content on screen may be read.

## Acceptance Criteria
See above.

## Bug Severity
- 3 - Moderate (because of the inability to choose the date picker with voiceover or talkback enabled.

## Linked to Story
https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/15763

## Screen shot(s) and additional information
![image.png](https://images.zenhubusercontent.com/5f6a83c84093730d05b5cee6/62fd3e2e-a25a-4895-921a-7404912a1380)

**Comment by digitalken** [comment]
1/12/2021
1. Pass
2. Pass
3. FAIL - Android still unable to cause Android to act on date picker via VoiceAccess.
4. No iOS build yet
5. No iOS build yet
6. No iOS build yet
7. Pass

@komaly15 
Regarding 3 above. I can get the numbers to appear, but when I say either number 9 or number 10, nothing changes on the screen. It acts like it is tapping or acting on the number when I say it and makes a <ping> noise, but nothing happens.  Tried with three different Androids.  Sending this one back to you.

![image.png](https://images.zenhubusercontent.com/5f6a83c84093730d05b5cee6/b980ffd6-0ce-4418-a163-4b18f8d11eab)

**Comment by digitalken** [comment]
PASS, closing 1/15/2021.
